packages/arb-token-bridge-ui/src/components/TransactionHistory/PendingTransactions.tsx

```diff
@@ -0,0 +1,64 @@
+import { motion } from 'framer-motion'
+import { InformationCircleIcon } from '@heroicons/react/outline'
+import Loader from 'react-loader-spinner'
+import { MergedTransaction } from '../../state/app/state'
+import { isDeposit } from '../../state/app/utils'
+import { motionDivProps } from '../MainContent/MainContent'
+import { DepositCard } from '../TransferPanel/DepositCard'
+import { WithdrawalCard } from '../TransferPanel/WithdrawalCard'
+import { useNetworksAndSigners } from '../../hooks/useNetworksAndSigners'
+import { getNetworkName } from '../../util/networks'
+
+export const PendingTransactions = ({
+  transactions,
+  loading,
+  error
+}: {
+  transactions: MergedTransaction[]
+  loading: boolean
+  error: boolean
+}) => {
+  const {
+    l2: { network: l2Network }
+  } = useNetworksAndSigners()
+
+  return (
+    <div className="relative flex max-h-[500px] flex-col gap-4 overflow-auto rounded-lg bg-blue-arbitrum p-4">
+      {/* Heading */}
+      <span className="flex flex-nowrap items-center gap-x-3 whitespace-nowrap text-xl text-white">
+        {loading && (
+          <Loader type="TailSpin" color="white" width={20} height={20} />
+        )}
+        {getNetworkName(l2Network.chainID)} Pending Transactions
```

Comment:
why does it always shows the L2 network name here?

Reply:
This was one of the suggestions in the design doc- to be more explicit about which Networks are we showing transactions related to. 
The reason is we don't want users to assume that this panel will contain both One and Nova transactions. To see Nova txns, they will need to switch their networks from the top toggle. So until we don't come up with a graceful switch for that, or we don't handle ALL chain transactions in the panel, we are being a little explicit about it.